docker.go

```diff
@@ -473,15 +477,16 @@ func newDockerImageSource(img, certPath string, tlsVerify bool) (*dockerImageSou
 		return nil, err
 	}
 	var tr *http.Transport
-	if certPath != "" {
```

Comment:
This is not refactor, this a bug fix (https://github.com/projectatomic/skopeo/pull/30/commits/9b73007ba196e432b2a6009c5a7bbf675ce10d62 ). We need to go down this `if` if `!tlsVerify` to create a `tls.Config{InsecureSkipVerify: true}`, even if we do not have any certificates; without that, `!tlsVerify` is ignored.